### ISSUE TITLE
cyclic learning rate

### ISSUE THREAD

**Issue by dcornet** [GBeurier/DECON#2]
I may be missing something but I'm not confident with the clr implementation. It looks like every time/epoch there is no improvement in mse, it change the learning rate. With this behaviour it doesn't allow multiple weight update steps. To my understanding this is contradictory with the iterative nature of gradient descent.
I'll take another coffee :-)
